docs/reference/cli/apps.md

```diff
@@ -0,0 +1,192 @@
+# apps
+
+## apps
+
+List apps
+
+### Usage
+
+    convox apps
+
+### Examples
+
+    $ convox apps
+    APP          STATUS   RELEASE
```

Comment:
Let's try to stick with the same dummy data used across the docs. This means an app named `myapp` and a release named `RABCDEFGHI`

See elsewhere in the docs for more examples of dummy data.

Reply:
https://github.com/convox/convox/blob/dd859cb23c6a78fc042bc72360db7f83539e6042/docs/reference/app.md

Reply:
Other primitives docs have more examples